### ISSUE TITLE
Error with YOLO-World inference

### ISSUE THREAD

**Issue by tyemelya** [roboflow/inference#850]
### Search before asking

- [X] I have searched the Inference [issues](https://github.com/roboflow/inference/issues) and found no similar bug report.

### Bug

Hello.
When I try to reproduce an example of inference using YOLO-World model
I receive an error 

 File "/python3.11/site-packages/inference/models/yolo_world/yolo_world.py", line 5, in <module>
    import clip
ModuleNotFoundError: No module named 'clip'

I have installed version of inference ==0.29.0, inference-sdk ==0.29.0 in my environement

### Environment

inference: 0.29.0 
inference-sdk:0.29.0
python: 3.11

### Minimal Reproducible Example

import cv2
import supervision as sv

from inference.models.yolo_world.yolo_world import YOLOWorld

image = cv2.imread("image.jpeg")

model = YOLOWorld(model_id="yolo_world/l")
classes = ["person", "backpack", "dog", "eye", "nose", "ear", "tongue"]
results = model.infer("image.jpeg", text=classes, confidence=0.03)[0]

detections = sv.Detections.from_inference(results)

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by tyemelya** [comment]
I've solved this problem by pip install inference[clip].

**Comment by PawelPeczek-Roboflow** [comment]
Yeah, but the good idea may be that we explicitly warn about missing deps during the import
